ProcessLib/LIE/HydroMechanics/HydroMechanicsProcess.cpp

```diff
@@ -357,6 +373,91 @@ void HydroMechanicsProcess<GlobalDim>::computeSecondaryVariableConcrete(
     GlobalExecutor::executeMemberOnDereferenced(
         &HydroMechanicsLocalAssemblerInterface::computeSecondaryVariable,
         _local_assemblers, *_local_to_global_index_map, t, x, coupled_term);
+
+    // Copy displacement jumps in a solution vector to mesh property
+    // Remark: the copy is required because mesh properties for primary variables are set
+    // during output and are not ready yet when this function is called.
+    int g_variable_id = 0;
+    int g_global_component_offset = 0;
+    {
+        int global_component_offset_next = 0;
+        int global_component_offset = 0;
+        for (int variable_id = 0;
+             variable_id < static_cast<int>(this->getProcessVariables().size());
+             ++variable_id)
+        {
+            ProcessVariable& pv = this->getProcessVariables()[variable_id];
+            int const n_components = pv.getNumberOfComponents();
+            global_component_offset = global_component_offset_next;
+            global_component_offset_next += n_components;
+            if (pv.getName() != "displacement_jump1")
+                continue;
+
+            g_variable_id = variable_id;
+            g_global_component_offset = global_component_offset;
+            break;
+        }
+    }
+
```

Comment:
You can use global index to get entries in a partition. To do that, replace lines 402 to 410 with
```
    MathLib::LinAlg::setLocalAccessibleVector(x).
```
https://github.com/ufz/ogs/pull/1797/files#diff-256366315d35994c6a0aa9c937d1a74cR25